pyproject.toml

```diff
@@ -135,3 +135,60 @@ warn_unused_ignores = true
 enable_error_code = [
   "ignore-without-code",
 ]
+
+[tool.uv]
+package = true
+
+[tool.uv.sources]
+hypercorn = { git = "https://github.com/urllib3/hypercorn", rev = "urllib3-changes" }
+
+[dependency-groups]
```

Comment:
Since this is standardized by PEP 735, can we move those just below the `[project.optional-dependencies]` section?